<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YellowPageEdit</class>
 <widget class="QWidget" name="YellowPageEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YellowPageEdit</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout">
      <property name="margin">
       <number>1</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="addButton">
       <property name="text">
        <string>追加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="editButton">
       <property name="text">
        <string>編集</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeButton">
       <property name="text">
        <string>削除</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
